internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -852,6 +852,28 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 								string(containerservice.LicenseTypeWindowsServer),
 							}, false),
 						},
+						"gmsa_profile": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Resource{
+								Schema: map[string]*pluginsdk.Schema{
+									"enabled": {
```

Comment:
Can we not expose this field for toggling GMSA, as with the other blocks in the AKS resource and across most of the provider the behaviour should be if block is present -> enabled, if block is absent -> disabled.